docs/End-User-Guide.md

```diff
@@ -1,258 +1,430 @@
+    <img src="images/survey-creator-multiple-text-set-default-answer.png" alt="Survey Creator - Matrix cell type" width="80%">
 
-![Image of the survey output of internal `Matrix Logic`](images/survey-editor-logic-matrix.png)
+#### Panel
 
-Second, the `Boolean` from Column 1/Row 1 is set to `yes/true`, which makes columns 2-4  `Visible`. Notice how this only applies to Row 1.
+The Panel type is a container for other questions and panels. Use this type to group several questions or panels and control them all together.
 
-![Image of the survey output of internal `Matrix Logic`](images/survey-editor-logic-matrix1.png)
+#### Dynamic Panel
 
-Third, the `Boolean` from Column 1/Row 2 is set to `yes/true`, which makes columns 2-4 `Visible` in Row 2 as well as Row 1. Note that Columns 2-4 in Row 3 are still not `Visible`. 
+The Dynamic Panel type is a template panel that can contain multiple questions. Respondents can add and remove panels based on the template.
 
-![Image of the survey output of internal `Matrix Logic`](images/survey-editor-logic-matrix2.png)
+<img src="images/question-types-dynamic-panel.png" alt="Question types - Dynamic panel" width="50%">
 
+### Adorners
 
-## Additional Resources
+Adorners are design-surface controls for survey element manipulation. Use adorners to edit UI texts in place, reorder choices, duplicate and delete elements, specify whether an answer is required, and perform other actions on survey elements. Settings that you specify using adorners are synchronized with settings in the Property Grid, and vice versa.
 
-- [SurveyJS Support Forums](https://surveyjs.answerdesk.io/ticket/list) - useful for searching user questions on specific features
+Each element type has an associated set of adorners. The following image highlights adorners for a [Dropdown](#dropdown) question:
+
+<img src="images/survey-creator-dropdown-adorners.png" alt="Survey Creator - Adorners" width="50%">
+
+### How to: Add a question to the survey
+
+Drag and drop the desired question type from the Toolbox onto the design surface. Alternatively, you can click the **Add Question** button to add a [Single Input](#single-input) question. This button also displays a three-dots icon that allows you to change the type of inserted questions.
+
+<img src="images/survey-creator-add-question-button.png" alt="Survey Creator - Add Question button" width="50%">
+
+### How to: Add a page to the survey
+
+In the Property Grid, select **Survey**, expand the **Pages** category, and click the **Add New** button in the **Pages** editor:
+
+<img src="images/survey-creator-add-new-page.png" alt="Survey Creator - Add new page" width="40%">
+
+Alternatively, you can add a page on the design surface. The surface displays the skeleton of a new page at the bottom. To add a new page, you need to [add a question to it](#how-to-add-a-question-to-the-survey).
+
+### How to: Change the question type
+
+Use the element type selector on the question:
```

Comment:
What do you mean by "on the question"? For example, do you mean "in the question"?